tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -221,13 +224,24 @@ def create_methodologies_snapshot(self) -> None:
         required=False,
     )
 
+    ap.add_argument(
+        "--headed",
+        dest="headed",
+        help="Run the tests in headed mode",
+        type=bool,
+        required=False,
+    )
```

Comment:
`run_tests.py` uses the `--visual`  argument to signal that you want the UI tests to not run headless. Not saying that's a better word than `--headed`, but for consistency, I think I'd keep it the same.

You could change `run_tests.py` to use `--headed` - my only problem with that is that people with existing scripts to run tests will have to change them. Not a big hassle, but it would force that on people.

All that said, happy to go along with whatever you think is best.

Reply:
Ah of course. didn't cross my mind to name it `-v` or `--visual`. Yeah `--headed` is something I came across when working with other UI test frameworks. The `--headed` and `--headless` flags are a bit more distinct imo but I'm happy to rename it to `-v` / `--visual`